niworkflows/interfaces/utils.py

```diff
@@ -126,6 +126,8 @@ class GenerateSamplingReferenceInputSpec(BaseInterfaceInputSpec):
                              desc='force xform code')
     fov_mask = traits.Either(None, File(exists=True), usedefault=True,
                              desc='mask to clip field of view (in fixed_image space)')
+    resolution = traits.Either('native', Str, traits.Int, None, usedefault=True,
```

Comment:
This isn't used except for `'native'`. What's the intended function with `Str` and `Int`?

Reply:
Probably a boolean would be better. I'll change that.